jbcli/jbcli/cli/jb.py

```diff
@@ -439,39 +365,64 @@ def cleanup():
               help='run an SSH tunnel for redshift')
 @click.option("--ganesha", default=False, is_flag=True,
               help="Enable ganesha")
+@click.option("--core", default=False, is_flag=True,
+              help="Use local Fruition checkout with this image")
 @click.pass_context
-def start(ctx, env, noupdate, noupgrade, ssh, ganesha):
+def start(ctx, env, noupdate, noupgrade, ssh, ganesha, core):
     """Configure the environment and start Juicebox"""
     if dockerutil.is_running():
         echo_warning('An instance of Juicebox is already running')
         echo_warning('Run `jb stop` to stop this instance.')
         return
 
-    env = get_environment_interactively(env)
+    # A dictionary of environment names and tags to use
+    tag_replacements = {
+        "hstm-dev": "hstm-qa",
+        "hstm-newcore": "develop-py3",
+        "core": "develop-py3",
+        "dev": "develop-py3",
+        "stable": "master-py3"
+    }
+
+    env = get_environment_interactively(env, tag_replacements)
+    core_path = "readme"
+    core_end = "unused"
+
+
+    if "core" in env or core:
+        if os.path.exists("fruition"):
+            core_path = "fruition"
+            core_end = "code"
+        else:
+            print("Could not find Local Fruition Checkout, please check that it is symlinked to the top level of Devlandia")
+            sys.exit()
+    if env in tag_replacements.keys():
+        tag = tag_replacements[env]
+    else:
+        tag = env
 
     if not noupgrade:
         ctx.invoke(upgrade)
 
-    stash.put('current_env', env)
-    os.chdir("./environments/{}".format(env))
+    stash.put('current_env', tag)
+    env_dot = open(".env", "w")
+    env_dot.write(f"DEVLANDIA_PORT=8000\nTAG={tag}\nFRUITION={core_path}\nFILE={core_end}")
+    env_dot.close()
+
     environ = populate_env_with_secrets()
 
```

Comment:
I think writing an `.env` file is an unnecessarily indirect way of setting these environment variables. We can just pass them in the environment when starting docker-compose.

Is there somewhere else this .env file is being relied on?

```suggestion

    environ = populate_env_with_secrets()
    environ.update({
        "DEVLANDIA_PORT": "8000",
        "TAG": tag,
        "FRUITION": core_path,
        "FILE": core_end,
    })
```

Reply:
I'll try this out, seems a bit more elegant than what I was trying. we are also relying on it for turning hstm on but we could probably also do it that way as well.